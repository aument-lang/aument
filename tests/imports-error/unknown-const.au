import "./empty.au" as mod;
mod::a;