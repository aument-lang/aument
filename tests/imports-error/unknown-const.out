interpreter error(10) in -: unknown or unexported constant 'a'
1 | import "./empty.au" as mod;
